pom.xml

```diff
@@ -80,6 +84,26 @@
       <version>${hamcrest.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>io.vertx</groupId>
+      <artifactId>vertx-web-templ-thymeleaf</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.vertx</groupId>
+      <artifactId>vertx-junit5</artifactId>
+      <scope>test</scope>
```

Comment:
We've got some duplicated dependencies here. I think the duplicated ones are:
 - vertx-junit5
 - junit-jupiter
 - hamcrest-core

Reply:
Suspect I missed this in the rebase, will fix